apps/FeedReader.md

```diff
@@ -19,24 +19,22 @@ links:
     url: "https://jangernert.github.io/FeedReader/"
   - type: GitHub
     url: jangernert/FeedReader
-  - type: Launchpad
-    url: feedreader
   - type: Translations
     url : "https://www.transifex.com/dev-feedreader/feedreader/"
   - type: License
-    url: "https://bazaar.launchpad.net/~eviltwin1/feedreader/trunk-1/view/head:/COPYING"
+    url: "https://github.com/jangernert/FeedReader/blob/master/COPYING"
 
 screenshots:
   - FeedReader.png
 
 installation:
   - system: elementary
     info:
-    - version: 0.3
+    - version: Stable
```

Comment:
Oh, I see. The stable version of FeedReader supports both versions of the OS. But the development version requires Gtk+ >= 3.20, thus supporting neither.